CHANGELOG.md

```diff
@@ -9,7 +64,7 @@
  - iPhone 5s devices and later with iOS 11+ is required.
 
 ### Bug Fixes
- - Fixed issues causing the app to crash on some devices
+ - Fixed issues causing the app to crash on some devices.
```

Comment:
If we cannot explicitly say which ones are these (that would make sense to the user), then I would say we should not add it

Reply:
I believe we had more bug fixes, not only around crashes.